pandora-client-web/src/graphics/graphicsCharacter.tsx

```diff
@@ -76,6 +79,10 @@ export const CHARACTER_PIVOT_POSITION: Readonly<PointLike> = {
 	y: 1290, // The position where heels seemingly touch the floor
 };
 
+const MIN_BLINK_INTERVAL = 4_000; // ms
+const MAX_BLINK_INTERVAL = 6_000; // ms
+const BLINK_LENGTH = 250; // ms
```

Comment:
Idea: Should the length of the blink be randomized too? Humans blink for 0.1 to 0.4 apparently, and I think randomizing this too might make it look more natural.

Reply:
I had wondered earlier if it should be or not, but since it added not entirely necessary complexity outside of the given spec, I decided against it for now. Since it's a relatively easy change though, it is now randomized as well.